bin/ci/before_install.sh

```diff
@@ -1,6 +1,8 @@
 #!/bin/bash
 set -v
 
+git config --global user.name "ManageIQ"
+git config --global user.email "contact@manageiq.org"
```

Comment:
why do we need these 2 `git config` commands?  wouldn't this override a developer's local setting?

Reply:
Developers don't run `bin/ci/before_install.sh`, so it's probably ok.  It might be safer to only set it if it's not set already, just in case.

@bdunne Ran into something similar when playing with GitHub Actions.  I think if we switch CI we're going to need this anyway.

Reply:
oh I see - this applies to CI only.  Your call @Fryguy

Reply:
I'm fine with keeping it for now.